### PR TITLE
Telemetry shim test

### LINKED ISSUE #29081: Repeat Telemetry OpenTracing Shim tests
MpTelemetry 2.0 is currently in line with the shim version [1.35](https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-opentracing-shim/1.35.0). Repeats need to test this.

### PR DESCRIPTION
**Pull request by Yas-Franco**
Fixes #29081

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat

**Review by benjamin-confino — DISMISSED** [review]

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#libby 
#build

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryShimTest.java [hunk 1]
```diff
@@ -58,21 +57,29 @@ public class TelemetryShimTest extends FATServletClient {
 
     @BeforeClass
     public static void setup() throws Exception {
-        if (RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP61_ID)) {
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryShimTest.java:61]: My only suggestion would be adding a comment like 

//Add the correct version of the shim library for each version of OTel.

But I will not block the merge for that.

**Review by benjamin-confino — DISMISSED** [review]

**Review by benjamin-confino — APPROVED** [review]

**Comment by Yas-Franco** [comment]
#build

**Pull request merged**